lib/crypto/src/signature/error.rs

```diff
@@ -0,0 +1,46 @@
+//! Signature error type.
+
+use core::fmt::{self, Debug, Display};
+
+// TODO: Once we update `alloy` to a crates.io version, use the `signature`
```

Comment:
IMHO we should have an issue for that.

Reply:
Actually, I realize this comment is wrong, since we don't want to depend on `alloy` here. I'll remove it.